src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/EqualsAvoidNullCheck.java

```diff
@@ -38,23 +38,8 @@
  * </p>
  * <p>Rationale: Calling the {@code equals()} method on String literals
  * will avoid a potential {@code NullPointerException}. Also, it is
- * pretty common to see null checks right before equals comparisons,
- * which is not necessary in the example below.
+ * pretty common to see null checks right before equals comparisons.
```

Comment:
`pretty common to see null checks right before equals comparisons but following this rule such checks are not required`.

Reply:
@romani Done